samples/bluetooth/radio_coex_3wire/src/main.c

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stddef.h>
+#include <sys/printk.h>
+#include <console/console.h>
+#include <sys/__assert.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/uuid.h>
+
+#include <hal/nrf_radio.h>
+#include <helpers/nrfx_gppi.h>
+#include <nrfx_timer.h>
+#include <nrfx_gpiote.h>
+#include <hal/nrf_egu.h>
+#include <nrfx_ppi.h>
+
+#define gppi_channel_t        nrf_ppi_channel_t
+#define gppi_channel_alloc    nrfx_ppi_channel_alloc
+#define gppi_channel_enable   nrfx_ppi_channel_enable
+#define gppi_channel_disable  nrfx_ppi_channel_disable
+#define APP_REQ_GPIO_PIN      28 // input to app
+#define APP_GRANT_GPIO_PIN    26 // output from app
```

Comment:
I think that ease of use would be improved if you allowed users to specify pins via devicetree in the `/zephyr,user` node.

See the `signal-gpios` example here for more details on how this works:

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/dts/bindings.html#inferred-bindings

You can convert this to a gpio_dt_spec or PSEL value in the sample using the same techniques I've mentioned elsewhere.

Reply:
I'll add it to the board overlay and readme. It's only being used for the 'dac' sample right now if I'm not mistaken.